sticky_notes.py

```diff
@@ -81,3 +81,37 @@ def list_stickynotes(self, event):
                 self.delay_task(index,
                                 self.sender(format_note(index, note), event))
             return first_note
+
+    @cmd
+    def del_note(self, event):
+        args = event.args
+        from_nick = event.meta['nick']
+
+        if len(args) < 1:
+            return "Missing nick argument."
+
+        if len(args) < 2:
+            return "Missing note number argument."
+
+        if len(args) >= 2:
+            nick = args[0]
+            note_num = args[1]
+
+            if nick in self.notes:
+                notes = self.notes[nick]
+                print len(self.notes[nick])
+                to_pop = -2
+
+                if note_num == 'last':
+                    if from_nick == notes[-1].sender:
+                        to_pop = -1
+
+                if note_num.isdigit() and int(note_num) <= len(notes):
+                    if from_nick == notes[int(note_num)].sender:
+                        to_pop = int(note_num)
+
+                if to_pop != -1:
```

Comment:
@JakubNvk again, why `to_pop != -1`?  What if `note_num` was `'last'`? It doesn't seem that the last note would be popped in that case.